packages/server/modules/workspaces/tests/integration/repositories/users.spec.ts

```diff
@@ -0,0 +1,292 @@
+import { db } from '@/db/knex'
+import {
+  createRandomEmail,
+  createRandomString
+} from '@/modules/core/helpers/testHelpers'
+import { getFeatureFlags } from '@/modules/shared/helpers/envHelper'
+import { getInvitableCollaboratorsByProjectIdFactory } from '@/modules/workspaces/repositories/users'
+import {
+  assignToWorkspace,
+  createTestWorkspace
+} from '@/modules/workspaces/tests/helpers/creation'
+import { createTestUser } from '@/test/authHelper'
+import { createTestStream } from '@/test/speckle-helpers/streamHelper'
+import { Roles } from '@speckle/shared'
+import { expect } from 'chai'
+import { pick } from 'lodash'
+
+const { FF_WORKSPACES_MODULE_ENABLED } = getFeatureFlags()
+
+;(FF_WORKSPACES_MODULE_ENABLED ? describe : describe.skip)(
```

Comment:
unnecessary, the entire workspace module tests get skipped if this FF is not set